chef-provisioning-docker.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/opscode/chef-provisioning-docker'
 
   s.add_dependency 'chef'
-  s.add_dependency 'chef-provisioning', '~> 0.9'
+  s.add_dependency 'chef-provisioning', '~> 1.0'
```

Comment:
Right. But I believe you want to force 1.2 or higher due to some fixes in 1.2 regarding chef-zero. @marc- mentioned that still didn't fix. I was gonna verify in my pr #52